wordpress-mobile/WordPress-iOS
# New Media Picker: auto-close the inline media picker during a multitasking pane resize

Fixes https://github.com/wordpress-mobile/MediaPicker-iOS/issues/201 (for now)

This PR will auto-close the input media picker if a multitasking pane resize occurs while the input picker is displayed. Not an ideal solution, but prevents the user from seeing an empty grey rect as described in the bug above ☝️.

![ipad-multi-issue](https://user-images.githubusercontent.com/154014/29833228-5cf2b26e-8cb0-11e7-80b1-3e1041371e08.gif)

Note that if the full-screen picker is visible while the multitasking pane is resized, hitting cancel should display the input picker as expected:

![ipad-multi-issue2](https://user-images.githubusercontent.com/154014/29833280-80764c32-8cb0-11e7-95fc-d6f0ca11a64c.gif)

**To test:**
On an iPad device/Sim (9.7" and 10.5"):
1. Edit a post and display the inline picker.
2. Resize the multitasking pane. 
3. Verify the input picker resigns and the normal text keyboard+toolbar is displayed.

In addition, try a variety of other multitasking interactions to see if any unexpected behaviors occur. Finally, test the inline picker on the iPhone to verify no regressions are introduced.

Needs review: @frosty ...would you mind?
